unit_tests/ut_h_reduce_data.amlg

```diff
@@ -215,5 +220,18 @@
     ))
     (call exit_if_failures (assoc msg "reduce_data with conservative thresholds reduced as expected"))
 
+	(print "a basic train automatically calls reduce_data when the threshold is low enough, resulting in min. model size w/ no threshold maps: ")
+    (call !EmptyAndTrain (assoc enable_auto_ablation (true) minimum_num_cases_to_reduce_data 120))
+    (call assert_same (assoc
+        exp
+            (get
+                (call_entity "howso" "get_auto_ablation_params")
+                (list 1 "payload" "minimum_num_cases")
+            )
+        obs
+            (get (call_entity "howso" "get_num_training_cases") (list 1 "payload" "count"))
```

Comment:
This test is failing because the actual number of training cases returned is 48 (which is also true in this test if you call `#train` with auto-ablation enabled but the `reduce_data` threshold too high for it to activate). Perhaps some interference from the ablation? I'm trying to figure out a better way to test this.